en/docs/quick-starts/oidc-session-management.md

```diff
@@ -0,0 +1,126 @@
+# Manage User Sessions and Logout
+
+This page guides you through managing user sessions and logout of a web application using a **sample application** called Playground.
+
+----
+If you have your own application, click the button below.
+
+<a class="samplebtn_a" href="../../guides/login/webapp-oidc"   rel="nofollow noopener">I have my own application</a>
+
+----
+{!fragments/register-a-service-provider.md!}
```

Comment:
this is the generic fragment for registering a service provider....since this is using the playground sample, shouldn't we include the full fragment 'oauth-playground' here so that we specify registering the sp for the playground app with the required sample callback URL

Reply:
oauth-playground does not contain information on Session Iframe Endpoint etc. So we can't use it

Reply:
session iframe info is given from the point where the user uses authorization code grant

the oauth-playground.md file only contains info up to the point where we register an sp for playground with the playground callback URL, and deploy the playground app.